nilearn/datasets/atlas.py

```diff
@@ -576,3 +576,73 @@ def fetch_yeo_2011_atlas(data_dir=None, url=None, resume=True, verbose=1):
         url=url,
         resume=resume,
         verbose=verbose)
+
+
+def fetch_atlas_aal(data_dir=None, url=None, resume=True, verbose=1):
+    """Download and return file names for the AAL for SPM 12 parcellation.
```

Comment:
It does not download file names ;). Can you just put `Downloads and returns the AAL template for SPM 12.`